UM-USElab/gradecraft-development
# Student index improvements

### Status
**READY**

### Description
Refactors view as a front-end component in Angular, for async and batch loading.

### Steps to Test or Reproduce
Observe that when you visit the see all students page, all of them load in batches with their intended information. Search and team filters should also work properly.

### Impacted Areas in Application
Students index page

======================
Addresses performance concerns for student index, as referred to in #3880
